setup.py

```diff
@@ -44,10 +44,10 @@ def run(self):
     install_requires=REQUIRES,
     packages=find_packages(),
     include_package_data=True,
-    cmdclass={
-        'clean': CleanCommand,
+    cmdclass={	
+        'clean': CleanCommand,	
     },
     long_description="""\
-    The DocuSign REST API provides you with a powerful, convenient, and simple Web services API for interacting with DocuSign.
+    The DocuSign REST API provides you with a powerful, convenient, and simple Web services API for interacting with DocuSign.  # noqa: E501
```

Comment:
is this an intentional addition?

Reply:
Yup that was part of ignore line for comments... new changes incorporated from open-source.